crypto/hrss/hrss.c

```diff
@@ -913,20 +914,26 @@ void HRSS_poly3_invert(struct poly3 *out, const struct poly3 *in) {
 // Coefficients are ordered little-endian, thus the coefficient of x^0 is the
 // first element of the array.
 struct poly {
+  uint16_t v[N+3];
```

Comment:
alignas(16) should be added here, there is a comment in the deleted code below:
`
// Even if !HRSS_HAVE_VECTOR_UNIT, external assembly may be called that requires alignment.
`